src/common/models/Projection.ts

```diff
@@ -257,6 +257,7 @@ export class Projection {
       summaryWithTimeseries.actuals,
       metrics,
       metricsTimeseries,
+      summaryWithTimeseries.population,
```

Comment:
This is actually stored in the class as `this.population` so you don't need to pass it in.

Reply:
Thank you for the tip. Done in da48bf914